Add SaDi-BRo/openai-bot (#39)
grammyjs/awesome-grammY: README.md

- [vrumger/WhisperedBot](https://github.com/vrumger/WhisperedBot) - Send whispers to people in groups so no one else can read them. [ᴜsᴇ](https://t.me/WhisperedBot)
- [voxelin/unihook](https://github.com/voxelin/unihook) - This bot makes a webhook listener and parses its payload by user-given template. [ᴜsᴇ](https://t.me/uniwebhookbot)
- [voxelin/groombot](https://github.com/voxelin/groom) - Groombot is a temporary email service that allows you to generate unlimited email addresses with your own domain on your phone. [ᴜsᴇ](https://t.me/groomtempbot)

> - [SaDi-BRo/openai-bot](https://github.com/SaDi-BRo/openai-bot) - OpenAI bot, which is built with official OpenAI's API, is made to use AI easily. (includes generate an image) [ᴜsᴇ](https://t.me/openaiuz_robot)

Reviewer: Make the description shorter and more straightforward.

- [vrumger/WhisperedBot](https://github.com/vrumger/WhisperedBot) - Send whispers to people in groups so no one else can read them. [ᴜsᴇ](https://t.me/WhisperedBot)
- [voxelin/unihook](https://github.com/voxelin/unihook) - This bot makes a webhook listener and parses its payload by user-given template. [ᴜsᴇ](https://t.me/uniwebhookbot)
- [voxelin/groombot](https://github.com/voxelin/groom) - Groombot is a temporary email service that allows you to generate unlimited email addresses with your own domain on your phone. [ᴜsᴇ](https://t.me/groomtempbot)

> - [SaDi-BRo/openai-bot](https://github.com/SaDi-BRo/openai-bot) - OpenAI bot, is made to use AI easily. (includes generate an image) [ᴜsᴇ](https://t.me/openaiuz_robot)

Reviewer: How about this?

```suggestion
- [SaDi-BRo/openai-bot](https://github.com/SaDi-BRo/openai-bot) - Interact with OpenAI APIs from Telegram. [ᴜsᴇ](https://t.me/openaiuz_robot)
```

Author: That's cool)
